man/systemd-bsod.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+
+<refentry id="systemd-bsod" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
this needs also `conditional='HAVE_QRENCODE'`

Reply:
> this needs also `conditional='HAVE_QRENCODE'`

ohh....

Reply:
Done sir!